examples/chip-tool/commands/common/CredentialIssuerCommands.h

```diff
@@ -57,10 +57,14 @@ class CredentialIssuerCommands
      *                        Verifier.
      * @param[in] trustStore  A pointer to the PAA trust store to use to find valid PAA roots.
      *
+     * @param[in] revocationDelegate A pointer to the Device Attestation Revocation Delegate for checking revoked DACs and PAIs.
+     *                               Default is nullptr.
+     *
      * @return CHIP_ERROR CHIP_NO_ERROR on success, or corresponding error code.
      */
-    virtual CHIP_ERROR SetupDeviceAttestation(chip::Controller::SetupParams & setupParams,
-                                              const chip::Credentials::AttestationTrustStore * trustStore) = 0;
+    virtual CHIP_ERROR
+    SetupDeviceAttestation(chip::Controller::SetupParams & setupParams, const chip::Credentials::AttestationTrustStore * trustStore,
```

Comment:
Please do not use default params for this, just require the value to always be passed.

Reply:
not using the default value.